src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -562,7 +586,65 @@ g.test('pipeline_output_targets,format_blendable')
 
 g.test('pipeline_output_targets,blend_min_max')
   .desc(`If the blend operation is "min" or "max", srcFactor and dstFactor must be "one".`)
-  .unimplemented();
+  .params(u =>
+    u
+      .combine('isAsync', [false, true])
+      .combine('component', ['color', 'alpha'] as const)
+      .beginSubcases()
+      .combine('srcFactor', kBlendFactors)
+      .combine('dstFactor', kBlendFactors)
+      .combine('operation', kBlendOperations)
+  )
+  .fn(async t => {
+    const { isAsync, component, srcFactor, dstFactor, operation } = t.params;
+
+    const defaultBlendComponent: GPUBlendComponent = {
+      srcFactor: 'src-alpha',
+      dstFactor: 'dst-alpha',
+      operation: 'add',
+    };
+    const blendComponentToTest = {
+      srcFactor,
+      dstFactor,
+      operation,
+    };
+    const fragmentShaderCode = t.getFragmentShaderCode('float', 4);
+    const format = 'rgba8unorm';
+
+    const descriptor =
+      component === 'color'
+        ? t.getDescriptor({
+            targets: [
+              {
+                format,
+                blend: {
```

Comment:
nit: we could do the ternary on blend itself, the rest of the descriptor seems identical? Or build it outside of this descriptor.